go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -56,20 +58,20 @@ func (bnc *BaseNetworkController) allocatePodIPs(pod *kapi.Pod,
 				"condition. Pod: %s/%s, node: %s", pod.Namespace, pod.Name, pod.Spec.NodeName)
 		}
 	}
-	if err := bnc.waitForNodeLogicalSwitchSubnetsInCache(switchName); err != nil {
+	if err := bnc.waitForNodeLogicalSwitchSubnetsInCache(switchNames.Original); err != nil {
 		return expectedLogicalPortName, fmt.Errorf("failed to wait for switch %s to be added to cache. IP allocation may fail!",
-			switchName)
+			switchNames)
```

Comment:
For non IC we should do nothing on that case (live migration and original node removed) for IC with one node per zone nothing will happend since the switchManager do not have that switch there.

So I will check if the node is still there and if not skip this, make sense ?

Reply:
If original and current are different and original is gone we should not return an error

Reply:
We should not run allocatedIPs if the original node is not at controller's local zone nodes, will do so.

Reply:
It's done at the point to point routing commit 
```
if kubevirt.IsPodLiveMigratable(pod) {                                  
            vm := kubevirt.ExtractVMNameFromPod(pod)                            
            if vm != nil {                                                      
                vms[*vm] = oc.isPodScheduledinLocalZone(pod)                    
            }                                                                   
            switchNames, err := oc.getSwitchNames(pod)                          
            if err != nil {                                                     
                return fmt.Errorf("failed reading vm pod %s/%s switch names at syncPods: %v", pod.Namespace, pod.Name, err)
            }                                                                   
            // after live migration the original node should fill in it's       
            // switch manager cache with VMs IP at controller restart           
            if _, ok := oc.localZoneNodes.Load(switchNames.Original); !ok {     
                continue                                                        
            }                                                                   
        } else if !oc.isPodScheduledinLocalZone(pod) {                          
            continue                                                            
        }                     
```